packages/react-hooks/src/useOwnStashInfos.ts

```diff
@@ -94,11 +104,16 @@ export function useOwnStashInfos (): StakerState[] | undefined {
   }, [api, mountedRef, ownStashes]);
 
   return useMemo(
-    () => ownStashes && queried && ownStashes.length === Object.keys(queried).length
-      ? ownStashes
-        .filter(([stashId]) => queried[stashId])
-        .map(([stashId]) => getStakerState(stashId, allAccounts, queried[stashId]))
-      : undefined,
-    [allAccounts, ownStashes, queried]
+    () => {
+      return {
+        isLoadingStashes: isLoading,
```

Comment:
Not sure this is needed - if 0, then nothing is loaded and we actually don't want/need payouts), it > 0 length, it is loaded. So would just forgo this.

Reply:
I followed `useOwnEraRewards` return structure. But I understand the trick of undefined vs [] to indicate loading state. I'd keep this for clean communication of intention, as `areAccountsLoaded` or that one in rewards.